tensorflow_datasets/core/dataset_builder.py

```diff
@@ -28,11 +28,16 @@
 
 from absl import logging
 import six
-import tensorflow.compat.v2 as tf
+import tensorflow.compat.v2 as tf  # type: ignore
```

Comment:
Why is `  # type: ignore` required ?

Reply:
giving below error in pytype:
```
Can't find module 'tensorflow'. [import-error]
```

Reply:
Did you install tensorflow on the same environment pytype is running ? It should works. If not, there is probably an issue in your set-up. Can you import Tensorflow in a simple script ?

Reply:
ok sure will try and ping u back.

Reply:
Thank you. Once this works, you can remove all the `# ptype: disable attribute-error` bellow.

Reply:
Done